packages/ui/components/overlays/test/OverlayController.test.js

```diff
@@ -229,47 +229,112 @@ describe('OverlayController', () => {
         await ctrl.show();
         // @ts-expect-error find out why config would/could be undfined
         expect(ctrl.content.style.zIndex).to.equal(`${ctrl.config.zIndex + 1}`);
+        expect(ctrl.manager.shownList).to.have.lengthOf(1);
```

Comment:
PR looks good to me. 
Any reason we should check for the shownList length here in these tests, instead of in the manager? https://github.com/ing-bank/lion/blob/master/packages/ui/components/overlays/test/OverlaysManager.test.js#L83